template/admin/initServer.sqf

```diff
@@ -10,6 +10,9 @@ private _setMissionName = {
     }, _this, _timeUntilStart] call CBA_fnc_waitAndExecute;
 };
 
+// Mission start notification
+[getMissionConfigValue ["onLoadName", ""], getText (configFile >> "CfgWorlds" >> worldName >> "description")] call _setMissionName;
```

Comment:
```suggestion
[getMissionConfigValue ["onLoadName", ""], getText (configFile >> "CfgWorlds" >> worldName >> "description")] call _setMissionName;
```
Don't some names have a version in them and so on?

Reply:
Only Lingor/Dingor & Panthera/Winthera that i know of, But i don't see it as much of a problem as the only alternate using `worldName` is worse in a lot of cases.

Some examples:
```
["Noe","Nogova"]
["pja305","G.O.S N'Ziwasogo"]
["pja307","G.O.S Dariyah"]
["pja308","G.O.S Gunkizli"]
["Sara","Sahrani"]
["Sara_DBE1","United Sahrani"]
["SaraLite","Southern Sahrani"]
["vt7","Virolahti"]
```

Reply:
Sounds good.

Reply:
```suggestion
[
    getMissionConfigValue ["onLoadName", ""],
    getText (configFile >> "CfgWorlds" >> worldName >> "description")
] call _setMissionName;
```

Reply:
Actually, this should just be moved to the event code itself. And `_setMissionName` doesn't have to be a function anymore and doesn't have to send any data over network.

```sqf
[{
    [QGVAR(missionName), []] call CBA_fnc_globalEvent;
}, [], _timeUntilStart] call CBA_fnc_waitAndExecute;
```